tests/Invoke-DbaAzSqlDbTip.Tests.ps1

```diff
@@ -0,0 +1,37 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        [object[]]$params = (Get-Command $CommandName).Parameters.Keys | Where-Object {$_ -notin ('whatif', 'confirm')}
+        [object[]]$knownParameters = 'SqlInstance', 'SqlCredential', 'LocalFile', 'Database', 'ExcludeDatabase', 'AllUserDatabases', 'ReturnAllTips', 'Compat100', 'StatementTimeout', 'EnableException', 'Force'
+        $knownParameters += [System.Management.Automation.PSCmdlet]::CommonParameters
+        It "Should only contain our specific parameters" {
+            (@(Compare-Object -ReferenceObject ($knownParameters | Where-Object {$_}) -DifferenceObject $params).Count ) | Should Be 0
+        }
+    }
+}
+
+Describe "$commandName Integration Tests" -Tags "IntegrationTests" {
+    if ($env:azuredbpasswd -eq "failstoooften") {
```

Comment:
@potatoqualitee - I copied this from the `Connect-DbaInstance` tests you added - but I'm not sure these actually ran - and wondering if I need to add this into a workflow? 🤔

Reply:
I wonder if we could use this to start testing the Az SQL commands in the emulator for Azure SQL DB?
Image that GitHub Actions should be able to pull down, so wouldn't use Appveyor.

Likely allow us to debug and run the tests on local machine as well 💕.

Reply:
https://learn.microsoft.com/en-us/azure/azure-sql/database/local-dev-experience-sql-database-emulator?view=azuresql

Reply:
Love that idea, but while testing it locally it seems like it's just a SQL server container? Either a standalone if you choose 'Full' or azure-sql-edge if you choose lite so the command fails.

![image](https://user-images.githubusercontent.com/981370/218483409-ad6c8af0-f5ad-47fb-a7cf-7d50de57d025.png)

Reply:
Yep, we will need to accept `azure-sql-edge` for connection (we should in our module anyway for those wanting to deploy it in their environment. Azure SQL Edge have the same bits for Azure SQL DB.

Reply:
Our GH actions (Linux, Mac and Windows) do connect to Azure successfully, you can put them in those integration tests